recipes/perl-hpc-runner-command/3.2.1/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# If it has Build.PL use that, otherwise use Makefile.PL
+git config --global user.email "you@example.com"
+git config --global user.name "Your Name"
+
+cpanm --installdeps .
```

Comment:
I'm still working on low level cpanm deps. I have another PR in from yesterday that gets some, and then I have quite a few more ready!